### ISSUE TITLE
Hyperlink creation automation feature requests

### ISSUE THREAD

**Issue by Leviob** [streetwriters/notesnook#4454]
### What problem are you facing?

Adding links requires clicking the toolbar and could be made more streamlined. 

### What's the solution you'd like to see in Notesnook?

- [ ] Adding a URL to a note and pressing space or return should automatically create a link to the url. This is the behavior in some text editors like Word. 
- [ ] When a URL is in the clipboard, selecting text and pasting should automatically create a link (rather than replacing the text) with the selected text as the link text and the URL from the clipboard as the link href. This is a (hidden?) feature of Slack when composing messages. 

### What alternatives have you considered?

Triggering the insert link popup with either a keyboard shortcut or clicking it and  automatically populating the clipboard URL in the link text field would be an okay compromise.

**Comment by thecodrr** [comment]
Both are done.

**Comment by Leviob** [comment]
Neither of these are working on Firefox 3.0.17-6def76e-web. 

It does work on Android 3.0.17, Thank you for adding this!

**Comment by thecodrr** [comment]
@Leviob tested on Firefox and it works. What exactly are you doing?

**Comment by Leviob** [comment]
I tried in Chrome on Ubuntu and it works perfectly, thank you. I tried on **Firefox on Windows** and it works too. 

However, in the latest **Firefox 130.0, on Ubuntu**, this does not work at all. It behaves the same as it did before where pasting a link just pastes the text and it does not turn into a hyperlink automatically. Highlighting text and pasting a link just replaces the originally highlighted text.
